docs/sources/static/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 canonical: https://grafana.com/docs/agent/latest/static/
 title: Static mode
+menuTitle: Static mode
```

Comment:
I'll remove for now. I added it because I had an initial thought of expanding the metadata at the same time as the other work here. Better that I have a dedicated PR for this metadata rework.